packages/esm-patient-common-lib/src/lab-resource/types/order.ts

```diff
@@ -0,0 +1,16 @@
+import { type Order } from '@openmrs/esm-patient-common-lib';
+
+export interface PatientMedicationFetchResponse {
+  results: Array<Order>;
+}
+
+export interface OrderDiscontinuationPayload {
+  previousOrder: string;
+  type: string;
+  action: string;
+  careSetting: string;
+  encounter: string;
+  patient: string;
+  concept: string;
+  orderer: { display: string; person: { display: string }; uuid: string };
```

Comment:
Is this the result we get back or the type for the message we send? Display strings should _never_ be sent to the backend under any circumstances.

Reply:
It should hold the `uuid` of the current practitioner for the user placing the order.